internal/services/environment_settings/resources_environment_settings.go

```diff
@@ -118,6 +118,14 @@ func (r *EnvironmentSettingsResource) Schema(ctx context.Context, req resource.S
 									boolplanmodifier.UseStateForUnknown(),
 								},
 							},
+							"log_retention_period_in_days": schema.Int64Attribute{
+								Description:         "Is audit retention period v2",
```

Comment:
Please change the description for something that the end user will understand idealy with a link to the documetnation

Reply:
Done.